diff --git a/src/components/KeyboardAvoidingView/index.tsx b/src/components/KeyboardAvoidingView/index.tsx
--- a/src/components/KeyboardAvoidingView/index.tsx
+++ b/src/components/KeyboardAvoidingView/index.tsx
@@ -162 +162 @@ const KeyboardAvoidingView = forwardRef<
-          return {};
+          return { height: undefined, flex: undefined };
diff --git a/lib/module/components/KeyboardAvoidingView/index.js b/lib/module/components/KeyboardAvoidingView/index.js
--- a/lib/module/components/KeyboardAvoidingView/index.js
+++ b/lib/module/components/KeyboardAvoidingView/index.js
@@ -86 +86,4 @@ const KeyboardAvoidingView = /*#__PURE__*/forwardRef(({
-        return {};
+        return {
+          height: undefined,
+          flex: undefined
+        };
diff --git a/lib/commonjs/components/KeyboardAvoidingView/index.js b/lib/commonjs/components/KeyboardAvoidingView/index.js
--- a/lib/commonjs/components/KeyboardAvoidingView/index.js
+++ b/lib/commonjs/components/KeyboardAvoidingView/index.js
@@ -94 +94,4 @@ const KeyboardAvoidingView = /*#__PURE__*/(0, _react.forwardRef)(({
-        return {};
+        return {
+          height: undefined,
+          flex: undefined
+        };
